docs/developer-guide/syntax-reference.rst

```diff
@@ -939,6 +939,35 @@ example of converting a BIGINT into a VARCHAR type:
       WINDOW TUMBLING (SIZE 20 SECONDS)
       GROUP BY page_id;
 
+CASE
+~~~~
+
+ **Synopsis**
+
+ .. code:: sql
+
+     CASE
+        WHEN condition THEN result
+        [ WHEN ... ]
```

Comment:
Can we change this to `[ WHEN ... THEN ... ]` to make it more clear that each `WHEN` must be paired with a `THEN`?

Reply:
Sure, added `THEN` too.